components/dashboard/src/workspaces/DeleteWorkspaceModal.tsx

```diff
@@ -33,7 +33,7 @@ export const DeleteWorkspaceModal: FunctionComponent<Props> = ({ workspace, onCl
             areYouSureText="Are you sure you want to delete this workspace?"
             children={{
                 name: workspace.id,
-                description: workspace.description,
+                description: workspace.name,
```

Comment:
It appears to work correct, I'm wondering though about "name vs description" change here. On the WS, I don't even see a description property. Must be some protocol mapping thing, right? 
(Just trying to understand the changes.)
<img width="344" alt="Screenshot 2023-11-16 at 13 00 48" src="https://github.com/gitpod-io/gitpod/assets/914497/3a2db326-3acd-4267-a3a9-7b7d71df5e56">

Reply:
We change description to name, see public-api-conver